<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Pardus için Cd/DVD Arşivleyici</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>39</y>
     <width>668</width>
     <height>510</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QSplitter" name="splitter_2" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="maximumSize" >
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title" >
        <string>Arşiv Adları</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QCheckBox" name="CbHepsiniAra" >
          <property name="font" >
           <font>
            <pointsize>7</pointsize>
           </font>
          </property>
          <property name="text" >
           <string>Tüm Arşivlerde  Ara</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QTableView" name="arsivadlari" >
          <property name="maximumSize" >
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="cursor" stdset="0" >
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet" >
           <string notr="true" >selection-color: rgb(85, 255, 0);</string>
          </property>
          <property name="showGrid" >
           <bool>false</bool>
          </property>
          <property name="cornerButtonEnabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QComboBox" name="comboBox" />
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QGroupBox" name="groupBox_2" >
        <property name="title" >
         <string>Dosya/Dizin</string>
        </property>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string/>
           </property>
           <property name="pixmap" >
            <pixmap resource="ikonlar.qrc" >:/ikonlar/ikonlar/search.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="arama_kutusu" >
           <property name="styleSheet" >
            <string notr="true" >selection-background-color: rgb(85, 255, 0);</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="3" >
          <widget class="QTableView" name="tablo" >
           <property name="alternatingRowColors" >
            <bool>true</bool>
           </property>
           <property name="selectionMode" >
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior" >
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="showGrid" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QCheckBox" name="CbDosyaYolu" >
           <property name="font" >
            <font>
             <pointsize>7</pointsize>
            </font>
           </property>
           <property name="text" >
            <string>Sadece Dosya Adına Göre Ara</string>
           </property>
           <property name="checked" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox" >
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>140</height>
         </size>
        </property>
        <property name="title" >
         <string>Arşiv İşlemleri</string>
        </property>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Arşiv Adı:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2" >
          <widget class="QLineEdit" name="arsiv_adi" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Arşivlenecek Yer:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2" >
          <widget class="QLabel" name="label" >
           <property name="palette" >
            <palette>
             <active>
              <colorrole role="WindowText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>28</red>
                 <green>74</green>
                 <blue>239</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Light" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>133</red>
                 <green>160</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Midlight" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>80</red>
                 <green>117</green>
                 <blue>247</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Dark" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>14</red>
                 <green>37</green>
                 <blue>119</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Mid" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>18</red>
                 <green>49</green>
                 <blue>159</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="BrightText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>28</red>
                 <green>74</green>
                 <blue>239</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Shadow" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="AlternateBase" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>141</red>
                 <green>164</green>
                 <blue>247</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>28</red>
                 <green>74</green>
                 <blue>239</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Light" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>133</red>
                 <green>160</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Midlight" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>80</red>
                 <green>117</green>
                 <blue>247</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Dark" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>14</red>
                 <green>37</green>
                 <blue>119</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Mid" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>18</red>
                 <green>49</green>
                 <blue>159</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="BrightText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>28</red>
                 <green>74</green>
                 <blue>239</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Shadow" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="AlternateBase" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>141</red>
                 <green>164</green>
                 <blue>247</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="WindowText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>14</red>
                 <green>37</green>
                 <blue>119</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>28</red>
                 <green>74</green>
                 <blue>239</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Light" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>133</red>
                 <green>160</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Midlight" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>80</red>
                 <green>117</green>
                 <blue>247</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Dark" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>14</red>
                 <green>37</green>
                 <blue>119</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Mid" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>18</red>
                 <green>49</green>
                 <blue>159</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>14</red>
                 <green>37</green>
                 <blue>119</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="BrightText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>14</red>
                 <green>37</green>
                 <blue>119</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>28</red>
                 <green>74</green>
                 <blue>239</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>28</red>
                 <green>74</green>
                 <blue>239</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Shadow" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="AlternateBase" >
               <brush brushstyle="SolidPattern" >
                <color alpha="255" >
                 <red>28</red>
                 <green>74</green>
                 <blue>239</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font" >
            <font>
             <weight>75</weight>
             <bold>true</bold>
             <underline>true</underline>
            </font>
           </property>
           <property name="text" >
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QPushButton" name="tbArsivYeri" >
           <property name="text" >
            <string>Arşivlenecek Dizin</string>
           </property>
           <property name="icon" >
            <iconset resource="ikonlar.qrc" >
             <normaloff>:/ikonlar/ikonlar/fileopen.png</normaloff>:/ikonlar/ikonlar/fileopen.png</iconset>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QPushButton" name="tbArsivYap" >
           <property name="text" >
            <string>Arşivi Oluştur</string>
           </property>
           <property name="icon" >
            <iconset resource="ikonlar.qrc" >
             <normaloff>:/ikonlar/ikonlar/ark.png</normaloff>:/ikonlar/ikonlar/ark.png</iconset>
           </property>
          </widget>
         </item>
         <item row="2" column="2" >
          <widget class="QPushButton" name="tbArsivSil" >
           <property name="text" >
            <string>Arşivi Sil</string>
           </property>
           <property name="icon" >
            <iconset resource="ikonlar.qrc" >
             <normaloff>:/ikonlar/ikonlar/stop.png</normaloff>:/ikonlar/ikonlar/stop.png</iconset>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <widget class="QPushButton" name="PbYedekle" >
           <property name="text" >
            <string>Yedekle</string>
           </property>
           <property name="icon" >
            <iconset resource="ikonlar.qrc" >
             <normaloff>:/ikonlar/ikonlar/filesave.png</normaloff>:/ikonlar/ikonlar/filesave.png</iconset>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QPushButton" name="PbGeriYukle" >
           <property name="text" >
            <string>Geri Yükle</string>
           </property>
           <property name="icon" >
            <iconset resource="ikonlar.qrc" >
             <normaloff>:/ikonlar/ikonlar/reload.png</normaloff>:/ikonlar/ikonlar/reload.png</iconset>
           </property>
          </widget>
         </item>
         <item row="3" column="2" >
          <widget class="QPushButton" name="pbHakkinda" >
           <property name="text" >
            <string>Hakkında</string>
           </property>
           <property name="icon" >
            <iconset resource="ikonlar.qrc" >
             <normaloff>:/ikonlar/ikonlar/misc.png</normaloff>:/ikonlar/ikonlar/misc.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>39</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground" >
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionArsivlenecek" />
   <addaction name="actionArsivolustur" />
   <addaction name="separator" />
   <addaction name="actionYedekle" />
   <addaction name="actionGeriyukle" />
   <addaction name="separator" />
   <addaction name="actionSil" />
   <addaction name="actionHakkinda" />
  </widget>
  <action name="actionArsivlenecek" >
   <property name="icon" >
    <iconset resource="ikonlar.qrc" >
     <normaloff>:/ikonlar/ikonlar/fileopen.png</normaloff>:/ikonlar/ikonlar/fileopen.png</iconset>
   </property>
   <property name="text" >
    <string>Arşivlenecek Dizin</string>
   </property>
   <property name="statusTip" >
    <string>Arşivlenecek Dizin</string>
   </property>
  </action>
  <action name="actionArsivolustur" >
   <property name="icon" >
    <iconset resource="ikonlar.qrc" >
     <normaloff>:/ikonlar/ikonlar/ark.png</normaloff>:/ikonlar/ikonlar/ark.png</iconset>
   </property>
   <property name="text" >
    <string>Arşivi Oluştur</string>
   </property>
   <property name="statusTip" >
    <string>Arşivi Oluştur</string>
   </property>
  </action>
  <action name="actionYedekle" >
   <property name="icon" >
    <iconset resource="ikonlar.qrc" >
     <normaloff>:/ikonlar/ikonlar/filesave.png</normaloff>:/ikonlar/ikonlar/filesave.png</iconset>
   </property>
   <property name="text" >
    <string>Arşivi Yedekle</string>
   </property>
   <property name="statusTip" >
    <string>Arşivi Yedekle</string>
   </property>
  </action>
  <action name="actionGeriyukle" >
   <property name="icon" >
    <iconset resource="ikonlar.qrc" >
     <normaloff>:/ikonlar/ikonlar/reload.png</normaloff>:/ikonlar/ikonlar/reload.png</iconset>
   </property>
   <property name="text" >
    <string>Arşivi Geri Yükle</string>
   </property>
   <property name="statusTip" >
    <string>Arşivi Geri Yükle</string>
   </property>
  </action>
  <action name="actionSil" >
   <property name="icon" >
    <iconset resource="ikonlar.qrc" >
     <normaloff>:/ikonlar/ikonlar/stop.png</normaloff>:/ikonlar/ikonlar/stop.png</iconset>
   </property>
   <property name="text" >
    <string>Arşivi Sil</string>
   </property>
   <property name="statusTip" >
    <string>Arşivi Sil</string>
   </property>
  </action>
  <action name="actionHakkinda" >
   <property name="icon" >
    <iconset resource="ikonlar.qrc" >
     <normaloff>:/ikonlar/ikonlar/misc.png</normaloff>:/ikonlar/ikonlar/misc.png</iconset>
   </property>
   <property name="text" >
    <string>Program Hakkında</string>
   </property>
   <property name="statusTip" >
    <string>Program Hakkında</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ikonlar.qrc" />
 </resources>
 <connections/>
</ui>
